server/src/types/group.types.ts

```diff
@@ -0,0 +1,23 @@
+import { Types } from 'mongoose';
+import { MongoInjectedFields } from './mongo.types';
+
+type GroupModel = {
+  groupName: string;
+  userIds: Types.Array<Types.ObjectId>;
+  setlistIds: Types.Array<Types.ObjectId>;
+  createdBy: string;
+  lastUpdatedBy: string;
+  createdAt: Date;
+  lastUpdatedAt: Date;
+  isDeleted: boolean;
+};
+
+type GroupList = {
```

Comment:
What's the purpose of this GroupList ya? I don't see it being used in other files

Reply:
ah yes unused. was testing something. probably for future